packages/esm-patient-chart-app/src/visit/visit-prompt/cancel-visit-dialog.test.tsx

```diff
@@ -96,20 +97,22 @@ describe('Cancel visit', () => {
 
     const cancelVisitButton = screen.getByRole('button', { name: /Cancel visit/i, exact: true });
 
-    await waitFor(() => user.click(cancelVisitButton));
+    // TODO: Will be fixed after the solution is finalized
```

Comment:
Ditto

Reply:
I left out the test cases to be added/ modified after finalizing the flows.

Reply:
Fixed the failing tests